SConstruct

```diff
@@ -1692,6 +1688,23 @@ imageEnvAppends = {
 	],
 }
 
+if env["PLATFORM"] != "win32" :
```

Comment:
I agree it gets confusing, I'll take another look at more elegant options. I like your idea of setting $ISYSTEM - as long as setting that to -I for Windows and -isystem for others works I expect I can drop the CPPPATH business and keep it cleaner.

Reply:
I sorted this by adding a new function that does the conditional in one place rather than each module. Formatting technicalities made the env["ISYSTEM"] method not work. Hopefully a nice side effect of the function is that it makes it a little more explicit what is going on.